tmp-frontend/lib/kinde_frontend_api/configuration.rb

```
@@ -0,0 +1,297 @@
+    end
+
+    # The default Configuration object.
+    def self.default
+      @@default ||= Configuration.new
+    end
+
+    def configure
+      yield(self) if block_given?
+    end
+
+    def scheme=(scheme)
```

RuboCop (1.76.1)

[warning] Method `KindeFrontendApi::Configuration#scheme=` is defined at both tmp-frontend/lib/kinde_frontend_api/configuration.rb:16 and tmp-frontend/lib/kinde_frontend_api/configuration.rb:180.

(Lint/DuplicateMethods)